python/basic-bot/README.md

```diff
@@ -1,83 +1,84 @@
 # Hangouts Chat basic bot
 
 This code sample creates a simple Hangouts Chat bot that responds to events and
-messages from a room. The sample is built using Python on Google App Engine,
+messages from a room. The sample is built using Python 3 on Google App Engine,
 Standard Environment.
 
+## Deploy the sample
+
+  1. Follow the steps in [Setting Up Your Development Environment](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment)
+     to install Python and the Google Cloud SDK
+
+  1. Follow the steps in [Setting Up Your GCP Resources](https://cloud.google.com/appengine/docs/standard/python3/console/#create)
+     to create a project and emable App Engine.
+
+  1. Rum the following command to deploy the app:
+     ```
+     gcloud app deploy
+     ```
+
+## Configure the bot for Hangouts Chat
+
+  1. To configure the bot to respond to @ mentions in Hangouts Chat, follow
+     the steps to enable the API in
+     [Publishing bots](https://developers.google.com/hangouts/chat/how-tos/bots-publish).
+  1. When configuring the bot on the **Configuration** tab on the
+     **Hangouts Chat API** page, enter the URL for the deployed version
+     of the bot into the **Bot URL** text box.
+
+
+## Interact with the bot
+
+Either add and @-mention the bot in a room or in a direct mention to engage with the bot.
```

Comment:
not picky if about using @-mention or @ mention, just consistency with line 22
looks like docs are inconsistent though and have @mention as well:
https://developers.google.com/s/results/hangouts/chat?q=mention

Reply:
Updated both to be @mention(s).